src/actions/org.js

```diff
@@ -64,8 +64,10 @@ const doSync = ({
   successMessage = 'Changes pushed',
   shouldSuppressMessages = false,
 } = {}) => (dispatch, getState) => {
-  const client = getState().syncBackend.get('client');
-  const path = getState().org.present.get('path');
+  const state = getState();
+  const presentOrgState = state.org.present;
```

Comment:
@schoettl I can absolutely understand why it looks like a good idea to minimize the duplication of typing `state.org.present` all the time! 

Having said that, to my knowledge, it's an anti-pattern to hide the structure of the reducers data shape. Maybe anti-pattern is too strong a word and maybe I'm wrong. But to the best of my knowledge, the data shape in a redux reducer is akin to a 'table' in SQL[1]. Hence, the same structure is repeated in all places in the code where a query happens. If we hide the structure here, it adds cognitive load that the structure in this scope is called differently than in other scopes.

Of course, I might be wrong and can be persuaded that this is indeed a good practice. Until then, I feel pretty certain it's better to repeat the structure in the code.

1. See 'key concepts': https://redux.js.org/recipes/structuring-reducers/prerequisite-concepts

Reply:
OK, I revert this. I understand the point.

Reply:
:pray: